sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -112,6 +112,11 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       checkFilterPredicate('_1 === true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 <=> true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 !== true, classOf[NotEq[_]], false)
+
+      checkFilterPredicate(('_1 in(true)).asInstanceOf[Predicate],
```

Comment:
BTW, I noticed that `In`, `StartsWith`, `EndsWith`, `Contains`, `RLike` and `Like` are all `Expression` but not `Predicate`. I don't see a good reason for this, and we can probably make all of them `Predicate`, but I may miss something here.